src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+  const LONG_TERM = 7;
+  const SHIRT_TERM = 3;
+  const LONG_TERM_DISKONT = 50;
+  const SHIRT_TERM_DISKONT = 20;
```

Comment:
There is a typo in the constant name 'SHIRT_TERM_DISKONT'. It should be 'SHORT_TERM_DISCOUNT' to maintain consistency and correct spelling.